### PR TITLE
feat: add webhook trigger for CI pipeline, add git trigger for invent…

### PR DESCRIPTION
**Pull request by brendankellyibm**
### Description

- Add webhook for CI pipeline
- Trigger that webhook at end of DA run to start the CI pipeline
- Git trigger for inventory to promote build once CI completes
- Will require inventory_repo_url as new parameter - take this from the ALM DA

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: solutions/banking/main.tf [hunk 1]
```diff
@@ -211,3 +211,60 @@ resource "ibm_cd_tekton_pipeline_property" "watsonx_assistant_integration_id_pip
...
+    ibm_cd_tekton_pipeline_property.watsonx_assistant_integration_id_pipeline_property_ci,
+    ibm_cd_tekton_pipeline_property.watsonx_assistant_id_pipeline_property_ci,
+    ibm_resource_instance.cd_instance
+  ]
+  triggers = {
+    always_run = timestamp()
+  }
+
+  provisioner "local-exec" {
+    command     = "${path.module}/watson-scripts/webhook-trigger.sh \"${ibm_cd_tekton_pipeline_trigger.ci_pipeline_webhook.webhook_url}\" \"${random_string.webhook_secret.result}\""
```
Comment (vburckhardt) [inline solutions/banking/main.tf:266]: This script seems to be missing from the PR

Comment (brendankellyibm) [reply solutions/banking/main.tf:266]: I missed that when copying all in from a test directory, included now.

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**